Module not found: Can't resolve 'antdesutton'

when `npm start` get the error:
``` bash
./src/components/Exception/index.tsx
Module not found: Can't resolve 'antdesutton' in 'C:\Users\ChiaJune\Desktop\poi-typescript\src\components\Exception'
```
OS: win10

--- a/lib/util.js
+++ b/lib/util.js
@@ -17,6 +17,14 @@ function tranCamel2(componentName, joinString) {
 }
 
 /**
+ * replace backward slash (\) with forward slash (/)
+ * @param {string} filePath path
+ */
+function winPath(filePath) {
+  return filePath.replace(/\\/g, '/');
+}
+
+/**
  * file exists in node_modules?
  * @param filePath
  */
@@ -72,9 +80,9 @@ function replaceImport(source, options = {}) {
           }
         }
       });
-      return ret;
+      return winPath(ret);
     } else {
-      return org;
+      return winPath(org);
     }
   });
 }

--- a/lib/util.js
+++ b/lib/util.js
@@ -21,7 +21,11 @@ function tranCamel2(componentName, joinString) {
  * @param {string} filePath path
  */
 function winPath(filePath) {
-  return filePath.replace(/\\/g, '/');
+  // for windows
+  if (process.platform === 'win32') {
+    return filePath.replace(/\\/g, '/');
+  }
+  return filePath;
 }
 
 /**
@@ -82,7 +86,7 @@ function replaceImport(source, options = {}) {
       });
       return winPath(ret);
     } else {
-      return winPath(org);
+      return org;
     }
   });
 }

